frontend/prod.Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM node:alpine as build
```

Comment:
It would be nice to order the commands to correspond w/ the dev.Dockerfile for consistency.

Reply:
We can, but they build two different kinds of images.  One for local development using webpack and another for production with nginx.

Reply:
I understand, but just for consistency's sake while someone's quickly skimming the dockerfiles, could you reorder this?

Reply:
> I understand, but just for consistency's sake while someone's quickly skimming the dockerfiles, could you reorder this?

Yes, I updated it to be near replicas.  The "as build" part is needed in the production file for the final image to use without having to build it every time; such as for an nginx config change.

Reply:
I guess what I mean is for them to be even closer, s.t. the order of the commands is the same when it can be i.e. `RUN webpack` after `RUN cp -a /tmp/node_modules /usr/src/app/` and `EXPOSE 80` after `COPY ./nginx/nginx.conf /etc/nginx/conf.d`. I'll admit this is a little nitpicky, but I think it would improve readability.